### PR TITLE
Add benchmarking scripts

### PR DESCRIPTION
**Pull request by zemdom**

### CODE REVIEW

**Review by mmatuszek1 — APPROVED** [review]
:lgtm:

> Reviewed 67 of 67 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/k8s-redis-cluster/3#-:-MqEPKdC414KCJg-f0Ri:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @adam-ad and @hsoltysi)

File: ansible-playbook/include/playbooks/process_memtier_results.yml [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+      shell: "grep \"{{ what_to_grep }}\" -A7 {{ client_results_dir }}/{{ run_id }}* | grep Totals | awk '{print $9}' | paste -sd+ | /usr/bin/bc | awk '{ total += $1 } END { print total }'"
+      register: p99_latency_stdout
+
+    - name: "process_memtier | Get total p99.9 latency"
+      shell: "grep \"{{ what_to_grep }}\" -A7 {{ client_results_dir }}/{{ run_id }}* | grep Totals | awk '{print $10}' | paste -sd+ | /usr/bin/bc | awk '{ total += $1 } END { print total }'"
+      register: p99_9_latency_stdout
+
+    - name: "process_memtier | Sum metrics values from all clients"
+      set_fact:
+        throughput: "{{ ansible_play_hosts_all | map('extract', hostvars, ['throughput_stdout', 'stdout']) | map('float') | sum | round(2) }}"
```
Comment (hsoltysi) [inline ansible-playbook/include/playbooks/process_memtier_results.yml:105]: In CONTRIBUTING.md doc, at Code Style section we daclare max. 90 chars per line... Do we have any justification here for breaking this rule? May we should update CONTRIBUTING.md?

Comment (zemdom) [reply ansible-playbook/include/playbooks/process_memtier_results.yml:105]: ansible-lint rule allows max. 160 chars per line, maybe we should update CONTRIBUTING.md to that value?

File: docker_bitnami/prebuildfs/opt/bitnami/scripts/libbitnami.sh [hunk 2]
```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+
+# Copyright (c) 2015-2021 Bitnami
```
Comment (hsoltysi) [inline docker_bitnami/prebuildfs/opt/bitnami/scripts/libbitnami.sh]: Are we aware of mixing licenses in src tree? And its consequences?

**Review by hsoltysi — CHANGES_REQUESTED** [review]
~20 files are introducing external Apache license granted by Bitnami. Are we aware of mixing license in src tree? Are we considered consequences (viral license etc.)?
In CONTRIBUTING.md doc, at Code Style section we daclare max. 90 chars per line... Do we have any justification here for breaking this rule? May we should update CONTRIBUTING.md?

File: docker_bitnami/prebuildfs/opt/bitnami/scripts/libbitnami.sh (hunk 2 above)
Comment (zemdom) [reply docker_bitnami/prebuildfs/opt/bitnami/scripts/libbitnami.sh]: Aware of mixing, but not sure about the consequences? If using the external files is not possible, could probably replace them with instruction where to find them, e.g here: https://github.com/bitnami/bitnami-docker-redis-cluster/tree/master/5.0/debian-10

File: ansible-playbook/include/playbooks/process_memtier_results.yml (hunk 1 above)
Comment (zemdom) [reply ansible-playbook/include/playbooks/process_memtier_results.yml:105]: CONTRIBUTING.md updated

**Review by hsoltysi — APPROVED** [review]

**Pull request merged**